src/assembler/parser/index.ts

```diff
@@ -79,7 +81,7 @@ export function parse(code: string): ICodeFile {
     },
     {
       name: 'Label',
-      pattern: `(${SYMBOL})(?=(?:${COMMENT}|\\s)*${SPACE_OR_TAB}${INSTRUCTION})`,
+      pattern: `(${SYMBOL})(?=(?:${COMMENT}|\\s)*(?:${SPACE_OR_TAB}${INSTRUCTION}|\\s(?:${LITERAL_SYMBOL_DECLARATION}|${AREA_DECLARATION})))`,
```

Comment:
Maybe also use a const varibale `LABEL_DECLARATION`

Reply:
like `const LABEL_DECLARATION = (${SYMBOL})` or what do you mean? Label is not used somewhere else yet probably with @deflox  bug for multiple labels it will get added

Reply:
Just like the others for `AREA` and `INSTRUCTION`.

`const LABEL_DECLARATION = (${SYMBOL})(?=(?:${COMMENT}|\\s)*(?:${SPACE_OR_TAB}${INSTRUCTION}|\\s(?:${LITERAL_SYMBOL_DECLARATION}|${AREA_DECLARATION})))`

Reply:
declared as const and added instruction without tab or space but newline so that not ParseError (unknown token ) is thrown if unindented instruction has a label on line before